app/2024/pyladies/page.jsx

```diff
@@ -0,0 +1,129 @@
+import Image from "next/image";
+import Link from "next/link";
+import { PYLADIES_REG } from "@/2024/utils/constants";
+
+export default function DjangoGirls() {
+  return (
+    <>
+      {" "}
+      {/* <section className="bg-fixed text-zinc-300" style={{ backgroundImage: "url('/assets/images/djangogirls/dg1.png')" }}> */}
+      {/* <section className="bg-fixed text-zinc-300 bg-pyconug-lightBlue" style={{}}>
+        <div className="flex flex-col justify-center items-center h-[70vh]">
+          <div>
+            <h1 className="text-3xl font-extrabold sm:text-5xl text-white">
+              Pyladies Anniversary
+            </h1>
+          </div>
+
+          <div>
+            {" "}
+            <p className="text-white p-4 font-medium w-[60vw] text-center">
+              We are pleased to inform you that the Pyladies will be celebrating their seventh Anniversary during the Pycon Uganda 2024 conference as a way to celebrate women and the achievements they've attained over time.
+            </p>
+            <p className="text-white p-4 font-medium text-center">
+              Kindly register here if you are interested in attending.
+            </p>
+          </div>
+        </div>
+      </section> */}
+      <section className="my-10 justify-center items-center text-justify text-lg">
+        <div className="mx-8 md:mx-28 max-w-screen-xl flex flex-col justify-center items-center mb-10">
+          <Image
+            width={150}
+            height={75}
+            alt=""
+            src={"/assets/images/pyladies/pyladies.png"}
+          />
+        </div>
+        <div className="mx-8 md:mx-28 max-w-screen-xl flex flex-col lg:flex-row mb-10">
+          <div className="lg:w-1/2 p-4 w-full">
+            <h1 className="text-3xl">PyLadies Anniversary</h1>
+            <p className="mt-3">Join us in celebrating the PyLadies Anniversary Event at PyCon Uganda!
+              This special occasion marks a significant milestone for PyLadies, an
+              international mentorship group dedicated to empowering women to become
+              active participants and leaders in the Python open-source community. Our mission
+              is to promote, educate, and advance a diverse Python community through
+              outreach, education, conferences, events, and social gatherings.</p>
+          </div>
+          <div className="lg:w-1/2 p-4 w-full">
+            <h1 className="text-3xl">Join Our Anniversary Workshop!</h1>
+            <p className="mt-3">Don&pos;t miss this chance to start your Python journey and join a supportive
```

Comment:
```suggestion
            <p className="mt-3">Don&apos;t miss this chance to start your Python journey and join a supportive
```
<img width="130" alt="image" src="https://github.com/PyconUg/ug.pycon.org/assets/65954740/154e04ac-21a8-469c-a93a-349d6ee40667">